File: benchmarks/jasboot_vs_v8/array_heavy_numeric.js
let acc = 0;
const datos = [];

for (let i = 0; i < 2048; i++) {
  datos.push((i * 7) % 997);
}

for (let i = 0; i < 300000; i++) {
  acc += (datos[i % 2048] * 3) + (i % 11);
}

console.log(acc);

File: benchmarks/jasboot_vs_v8/array_heavy_numeric.c
#include <stdio.h>

int main(void) {
    int datos[2048];
    long long acc = 0;

    for (int i = 0; i < 2048; i++) {
        datos[i] = (i * 7) % 997;
    }

    for (int i = 0; i < 300000; i++) {
        acc += (datos[i % 2048] * 3) + (i % 11);
    }

    printf("%lld\n", acc);
    return 0;
}
